src/applications/disability-benefits/526EZ/containers/ConfirmationPage.jsx

```diff
@@ -36,8 +36,21 @@ class ConfirmationPage extends React.Component {
       <div>
         <h3 className="confirmation-page-title">Claim submitted</h3>
         <p>We usually process claims within <strong>103 days</strong>.</p>
-        <p>We may contact you if we have questions or need more information. You can print this page for your records.</p>
-
+        <p>We may contact you if we have questions or need more information.
+        You can print this page for your records. <strong>If you need us to get
+        your private medical records from your doctor,</strong> you’ll need to
+        fill out an Authorization to Disclose Information to the VA (VA Form
+        21-4142). If you didn’t upload this form to your application, you can
+        download it here and mail it in.</p>
+        <a href="https://www.vba.va.gov/pubs/forms/VBA-21-4142-ARE.pdf" target="_blank">
+        Download VA Form 21-4142</a>
+        <p>
```

Comment:
Do we want to use `.va-address-block` here (perhaps a question for @goldenmeanie or @peggygannon)? It would produce the following address format:
![image](https://user-images.githubusercontent.com/16051172/40191664-04083bfe-59b7-11e8-9982-f18c57391669.png)

Reply:
That looks nice! Good find. Will wait for Alex / Peggy to confirm.

Reply:
@markgreenburg Yes, the VA block address format looks good to me!